climada/hazard/trop_cyclone.py

```diff
@@ -485,25 +485,17 @@ def _apply_knutson_criterion(self, chg_int_freq, scaling_rcp_year):
                 sel_cat_chg = np.isin(tc_cc.category, cat_chg_list) & bas_sel
                 if sel_cat_chg.any():
                     freq_scaling = 1 + (chg['change'] - 1) * scaling_rcp_year
-                    sel_cat_all = (np.isin(tc_cc.category, chg['category'])
-                                   & bas_sel)
-                    sel_cat_larger = (np.isin(tc_cc.category, cat_larger_list)
-                                      & bas_sel)
-                    freq_scaling_cor = (
-                        (np.sum(self.frequency[sel_cat_all]) * freq_scaling
-                         - np.sum(tc_cc.frequency[sel_cat_larger]))
-                        / np.sum(self.frequency[sel_cat_chg])
-                    )
-                    tc_cc.frequency[sel_cat_chg] *= freq_scaling_cor
+                    tc_cc.frequency[sel_cat_chg] *= freq_scaling
                 cat_larger_list += cat_chg_list
 
         if (tc_cc.frequency < 0).any():
             raise ValueError("The application of the given climate scenario"
                              "resulted in at least one negative frequenciy."
-                             "This is likely due to the use of a"
+                             "This is likely due to the use of a "
                              "non-representative event set (too small, "
```

Comment:
After reverting the changes, it is no longer true that negative frequencies would be due to a non-representative event set. Instead, this will happen for extremely high-emission scenarios in basins and categories with high reductions in frequency according to Knutson. For example, RCP 8.5 with reference year 2100 and 'WP' or 'SP' basin.

Reply:
1.  Correct, it was sloppy writing, but you get the point right? Instead of  making a subtraction, the scaling from climate change should be relative. Then, negative frequencies cannot happen.
2. Yes,  but if an error is discovered during this process it should be addressed right away, potentially leading to a new pull request.

Reply:
No, I'm not sure I get the point because I don't understand why we should inverse the meaning of the scaling factor. Maybe you can explain in a bit more detail?

Reply:
Ok, sorry for the lack or clarity. So, the scaling factor can become negative, but this is imho an artefact of a bad implementation. 

Suppose the climate change scaling factor is 2. Then:
- Knutson factor 110%=1.1 -> change  = 1+(1.1-1)*2 = 1.2
This looks reasonable, we go from an increase by 10% to 20%.
- Knutson factor 10%=1/10=0.1 -> change = 1+(0.1-1)*2 = -0.8
This is not  reasonable. What we should expect is rather to go from a reduction of 10% to 5%=1/20=0.05.

Reply:
What makes you think that more climate change will produce less reduction in frequency?

Reply:
I think the message was still unclear. Say frequency is 1, climate change factor 2. Then:

 - Knutson factor 110%=1.1 -> change = 1+(1.1-1)*2 = 1.2
This looks reasonable, we go from an increase by 10% (frequency 1.1) to 20% (frequency 1.2). 
- Knutson factor 10%=1/10=0.1 -> change = 1+(0.1-1)*2 = -0.8
This is not reasonable as frequency would be -0.8. What we should expect is rather to go from a reduction to 10% (frequency 0.1) to 5%=1/20=0.05 (frequency 0.05).

Reply:
As a possible solution (will  probably need another pull request), I propose the change in the comment below https://github.com/CLIMADA-project/climada_python/pull/224#discussion_r629339786